packages/react-core/src/components/OverflowMenu/__tests__/OverflowMenu.test.tsx

```diff
@@ -42,4 +43,41 @@ describe('OverflowMenu', () => {
 
     expect(myMock).toHaveBeenCalled();
   });
+
+  test('should call resizeObserver on undefined containerRefElement', () => {
+    const resizeObserver = jest.spyOn(getResizeObserver, 'getResizeObserver');
+
+    render(<OverflowMenu breakpoint={'md'} />);
+
+    expect(resizeObserver.mock.calls[1]).toEqual(undefined);
```

Comment:
Nit: per our discussion on Slack I think the usage of `.mock.calls` could be made a bit more readable using the `toHaveBeenNthCalledWith` matcher. This also applies to lines 68 and 81.